PermanentOrg/web-app
# PER-8427: Enable NR browser agent

## Description
This PR enables the New Relic Browser Agent on each environment (local, dev, staging, and prod).

I will be testing this branch on dev to see if the dev environment is properly set up.

Resolves PER-8427.

## Steps to Test
- Open the web-app
- Check New Relic to see if the corresponding Browser App has responded to the data.
